tools/CouchDB_MRI_Importer.php

```diff
@@ -99,12 +99,12 @@ function _generateCandidatesQuery($ScanTypes)
             $Query .= ", (SELECT f.File FROM files f LEFT JOIN mri_scan_type msc
               ON (msc.ID= f.AcquisitionProtocolID)
               WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype' LIMIT 1)
-                    as Selected_$scantype, (SELECT fqc.QCStatus
+                    as `Selected_$scantype`, (SELECT fqc.QCStatus
```

Comment:
Hey @ridz1208 Backticks aren't part of standard ANSI SQL. What if we move away from mysql?

Reply:
How are you suggesting the field name be quoted?

Reply:
@driusan I'm not sure whats best here. I was reading [stackoverflow](https://stackoverflow.com/questions/261455/using-backticks-around-field-names) about the topic of backticks and one person suggests to enable MySQL's ANSI mode for using double quotes instead but I'm not sure.

Reply:
I think there are other examples of backticks in the codebase. If we decide to support something other than MySQL/MariaDB then there will be many more changes to make. I think this is fine.